components/ai_chat/core/browser/engine/oai_api_client.cc

```diff
@@ -86,7 +87,9 @@ void OAIAPIClient::PerformRequest(
     GenerationDataCallback data_received_callback,
     GenerationCompletedCallback completed_callback) {
   if (!model_options.endpoint.is_valid()) {
-    std::move(completed_callback).Run(base::unexpected(mojom::APIError::None));
+    std::move(completed_callback)
```

Comment:
this feels like an error case

Reply:
actually, it might be better if we do an internal `CHECK` because this shouldn't happen

Reply:
Either way, things probably aren't going to work - I'm not sure about crashing in network related code, but doesn't a `CHECK` normally imply something serious is going on and you can't recover? In this case we can't recover, as nothing will work without a api endpoint

Reply:
makes sense. i have replaced it with `CHECK`